annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/tag/BodyTypesTest.java

```diff
@@ -0,0 +1,354 @@
+	static final String VALUE_DESCRIBING_TARGET_SCOPE = "target_uri:\"http://data.europeana.eu/item/07931/diglit_uah_m1\"";
+	static final String VALUE_BODY_URI = "body_uri:\"http://www.geonames.org/2988507\"";
+	static final String VALUE_DESCRIBING_BODY_VALUE = "body_value:\"... this is the textual description of the item ...\"";
+	static final String VALUE_BODY_VALUE = "body_value:\"trombone\"";
+	static final String VALUE_BODY_LINK_RELATION = "link_relation:\"isSimilarTo\"";
+	static final String VALUE_BODY_LINK_RESOURCE_URI = "link_resource_uri:\"http://thesession.org/tunes/52\"";
+	static final String VALUE_BODY_SPECIFIC_RESOURCE = "body_uri:\"http://www.geonames.org/2988507\""; // source
+	static final String VALUE_BODY_FULL_TEXT_RESOURCE = "body_value:\"... complete transcribed text in HTML ...\"";
+
+	static final int TOTAL_BY_ID_FOUND = 1;
+	
+	protected Logger log = LogManager.getLogger(getClass());
+	
+	private AnnotationSearchApiImpl annotationSearchApi;
+	
+	
+	/**
+	 * Create annotations data set before each test execution
+	 * 
+	 * @throws IOException
+	 */
+	@BeforeEach
+	public void createAnnotationSearchApi() throws JsonParseException, IOException {
+		annotationSearchApi = new AnnotationSearchApiImpl();
+	}
+	
+	@Test
+	public void createSemanticTag() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+		
+		String requestBody = getJsonStringInput(SEMANTICTAG_SIMPLE_STANDARD);
+		
+		Annotation inputAnno = parseTag(requestBody);
+		
+		// create indexed tag
+		Annotation storedAnno = createTag(requestBody);
+		
+		assertTrue(BodyInternalTypes.isSemanticTagBody(storedAnno.getBody().getInternalType()));
+		//validate the reflection of input in output!
+		validateOutputAgainstInput(storedAnno, inputAnno);
+		
+		// search for indexed id and textual values
+		searchByBodyValue(VALUE_ID+"\""+storedAnno.getAnnotationId().getIdentifier()+"\"", TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_BODY_URI, TOTAL_BY_ID_FOUND);
+		searchByBodyValue(VALUE_TARGET, TOTAL_BY_ID_FOUND);
+		
+		// validate fields
+		assertTrue(storedAnno.getMotivation().equals(MotivationTypes.TAGGING.name().toLowerCase()));
+		assertEquals(storedAnno.getBody().getInternalType(), BodyInternalTypes.SEMANTIC_TAG.name());
+		TagBody tagBody = ((SemanticTagBody) storedAnno.getBody());
+		assertNotNull(tagBody.getHttpUri());
+		assertTrue(tagBody.getHttpUri().equals("http://www.geonames.org/2988507"));
+		assertNotNull(tagBody.getLanguage());
+		assertTrue(tagBody.getLanguage().equals("en"));
+		Target target = storedAnno.getTarget();
+		assertNotNull(target.getHttpUri());
+		assertTrue(target.getHttpUri().equals("http://data.europeana.eu/item/09102/_UEDIN_214"));
+
+		// remove tag
+		deleteAnnotation(storedAnno);
+	}
+
+	@Test
+	public void createGeoTag() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
this is a search not a create test, please rename

Reply:
changed to search...